charts/optimize-live/templates/deployment.yaml

```diff
@@ -34,33 +35,28 @@ spec:
           envFrom:
           - secretRef:
               name: {{ include "optimize-live.fullname" . }}
-{{- if .Values.debug }}
           args:
+          - --config=/config/controller-manager.yaml
+          {{- if .Values.debug }}
           - --zap-log-level=9
-{{- end }}
+          {{- end }}
           env:
           - name: IMAGE_PULL_SECRET_NAME
             value: {{ include "optimize-live.fullname" . }}-docker
           - name: SF_CREDS_SECRET_NAME
             value: {{ include "optimize-live.fullname" . }}
 {{- if .Values.metricsURL }}
-{{- if (ne "" .Values.metricsURL) }}
```

Comment:
these are unrelated to PR, can break out to separate PR if preferred.  just done to make it a bit simpler to read.